ckanext/dalrrd_emc_dcpr/logic/action/dcpr/update.py

```diff
@@ -309,6 +309,7 @@ def create_package_from_dcpr_request(
                 data_dict["extras"] = [
                     {"key": "origin", "value": "DCPR"},
                     {"key": "action", "value": action.value},
+                    {"key": "status", "value": "completed"},
```

Comment:
@Samweli am changing the status of the dataset once created to completed for versioning reasons, we need to set this value to indicate that this dataset will not be overwritten in the future and only new versions of it would be created if any changes took place, for further info, please see the comments on issue #300 
if you have any inputs please go share them here.